examples/pedestal_noise_mc_analysis.xml

```diff
@@ -0,0 +1,75 @@
+<container>
```

Comment:
could you add a doc string that is telling what the process' purpose and what it is doing?

Reply:
BTW why is this process needed?

Reply:
Right now it is due to some errors.

Reply:
anyways, doc string please